cli/src/run_testenv.rs

```diff
@@ -44,6 +45,11 @@ pub struct RunTestenv {
     empty: bool,
 }
 
+pub fn process_id_to_port(pid: u32) -> u16 {
+    // The first ports are reserved, so let's take an available port
+    pid as u16 % AVAILABLE_PORT_COUNT + RESERVED_PORT_OFFSET
+}
```

Comment:
Where do this come from? There is no relation between pid and port :thinking: 
I did not found something similar in the v2 branch.

Also: modulo should be the last operation, otherwise the return port may be greater than the available port count.

Reply:
It comes from me to have an available port for the testbed server.
Note that there is a `RESERVED_PORT_OFFSET` which means that the port from 1 to 1024 are reserved.
The modulo guarantees that you can't overflow.

Reply:
Yes it avoids to store the port value, we are not using a constant value, because we run tests in multiple threads, so we will have multiple servers. I use the PID to retrieve this port in test, if you have another idea you are welcome.

Reply:
Done